src/pages/saksbehandling/sakintro/RevurderingsLister.tsx

```diff
@@ -0,0 +1,258 @@
+import { BodyShort, Heading, Label, Panel, Tag } from '@navikt/ds-react';
+import { last } from 'fp-ts/lib/Array';
+import { toNullable } from 'fp-ts/lib/Option';
+import Ikon from 'nav-frontend-ikoner-assets';
+import React from 'react';
+import { IntlShape } from 'react-intl';
+
+import LinkAsButton from '~components/linkAsButton/LinkAsButton';
+import UnderkjenteAttesteringer from '~components/underkjenteAttesteringer/UnderkjenteAttesteringer';
+import { useUserContext } from '~context/userContext';
+import { pipe } from '~lib/fp';
+import * as Routes from '~lib/routes';
+import { AbstraktRevurdering, RevurderingsStatus } from '~types/Revurdering';
+import { Sak } from '~types/Sak';
+import { Vedtak } from '~types/Vedtak';
+
+import {
+    erRevurderingTilAttestering,
+    erRevurderingIverksatt,
+    erRevurderingSimulert,
+    erForhåndsvarselSendt,
+    finnNesteRevurderingsteg,
+    erRevurderingStans,
+    erRevurderingGjenopptak,
+    erRevurdering,
+} from '../../../utils/revurdering/revurderingUtils';
+import { RevurderingSteg } from '../types';
+
+import { AvsluttOgStartFortsettButtons } from './Sakintro';
+import styles from './sakintro.module.less';
+
+export const ÅpneRevurderinger = (props: { sak: Sak; åpneRevurderinger: AbstraktRevurdering[]; intl: IntlShape }) => {
```

Comment:
Kunne man hentet formatMessage her i steden for å sende inn intl? 🤔  Eller er det en grunn til at man sender inn?

Reply:
intl tilbyr oss dato-formatering. Jeg mekka slik at man ikke nødvendigvis trenger intl for dato-formateringer. men på sikt, når man skal støtte andre språk (kanskje?) så må man bruke intl uansett.